packages/material-ui/src/Chip/Chip.js

```diff
@@ -272,7 +273,7 @@ const Chip = React.forwardRef(function Chip(props, ref) {
     className,
     clickable: clickableProp,
     color = 'default',
-    component: Component = 'div',
+    component: Component = ButtonBase,
```

Comment:
@oliviertassinari 
How about this change ?
component: Component = onClick && !onDelete ?ButtonBase:"div",

Reply:
It's worth trying?

Reply:
I think so